core.yml

```diff
@@ -86,7 +86,7 @@ Metrics/CyclomaticComplexity:
 
 Metrics/MethodLength:
   IgnoredMethods: ["extended"]
-  Max: 15
+  Max: 50
```

Comment:
50 seems excessive IMO. I'd argue 25-30 lines max, otherwise we're missing an opportunity to use these warnings to encourage teams to refactor / tend to codebase improvements.

Reply:
I think 25 seems like a reasonable number. I tested this out in `client-service` and found:

When I set `MethodLength` to 20, it solved for 11 out of the 22 places that we disabled it.
When I set `MethodLength` to 25, it solved for 17 out of the 22 places that we disabled it (and the other 5 did seem like exceptionally long methods.)
When I set `MethodLength` to 30, it solved for 18 out of the 22 places that we disabled it.